### PR TITLE
ref(models): Move more models to submodules

### PR DESCRIPTION
**Pull request by mgaeta**
In Backend TSC, we decided that it's time to start organizing some of our larger, flat directories. In this PR I move some additional models into the integrations directory.

### CODE REVIEW

File: src/sentry/models/integrations/__init__.py [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
+from .external_actor import ExternalActor
+from .external_issue import ExternalIssue
 from .integration import Integration
 from .integration_external_project import IntegrationExternalProject
+from .integration_feature import IntegrationFeature
 from .organization_integration import OrganizationIntegration
 from .pagerduty_service import PagerDutyService
 from .project_integration import ProjectIntegration
 from .repository_project_path_config import RepositoryProjectPathConfig
+from .sentry_app import SentryApp
```
Comment (mgaeta) [inline src/sentry/models/integrations/__init__.py:32]: I'd be ok with making a separate `/src/sentry/models/sentry_apps/` directory but I think it make sense to leave them with the other Integration models.

**Review by ceorourke — COMMENTED** [review]
Looks good to me, but I think you need to rebase.

**Review by NisanthanNanthakumar — APPROVED** [review]

File: src/sentry/incidents/serializers/alert_rule.py [hunk 2]
```diff
@@ -243,7 +243,6 @@ def validate(self, data):
             try:
                 raw_snql_query(snql_query, referrer="alertruleserializer.test_query")
             except Exception:
-                # TODO MARCOS 1
```
Comment (ceorourke) [inline src/sentry/incidents/serializers/alert_rule.py:246]: I think this was left in on accident

File: src/sentry/mediators/plugins/migrator.py [hunk 3]
```diff
@@ -5,7 +5,6 @@
 
 
 class Migrator(Mediator):
-    # TODO MARCOS FIRST
```
Comment (ceorourke) [inline src/sentry/mediators/plugins/migrator.py:8]: Also this

File: src/sentry/incidents/serializers/alert_rule.py (hunk 2 above)
Comment (mgaeta) [reply src/sentry/incidents/serializers/alert_rule.py:246]: This is a deletion.

**Pull request merged**